### DISCUSSION TITLE
Safari plugins on iphone?

### DISCUSSION THREAD

**Discussion by fredmo** [typefully/minimal-twitter#154] [Ideas]
Hello,

Is there a way to have this kind of plugin on iOs iPhone as an extension of Safari browser?

Regards,

**Comment by ghost** [comment]
Hi @fredmo good idea! The Typefully team is in the process of publishing the Safari extension. 

May be able to get it on both macOS and iOS with https://developer.apple.com/support/universal-purchase cc @linuz90
